libcxx/test/std/containers/sequences/deque/deque.modifiers/insert_range.pass.cpp

```diff
@@ -6,6 +6,10 @@
 //
 //===----------------------------------------------------------------------===//
 
+// FIXME: This takes over an hour to compile, disable for now.
+// UNSUPPORTED: target=amdgcn-amd-amdhsa
+// UNSUPPORTED: target=nvptx64-nvidia-cuda
```

Comment:
Can you instead create a `LIBCXX-NVPTX-FIXME` Lit feature that you can define in your CMake caches instead? It makes it easier to track these TODOs. You can look for `LIBCXX-PICOLIBC-FIXME` for a precedent.